t/Test28-fregrid_stretched.sh

```diff
@@ -0,0 +1,118 @@
+               --nlon 512 \
+               --grid_name C384_grid_35.4
+  [ "$status" -eq 0 ]
+
+
+#Create stretched grid mosaic
+  run command make_solo_mosaic \
+                --num_tiles 6 \
+                --dir ./ \
+                --mosaic_name C384_mosaic_32.0 --tile_file C384_grid_32.0.tile1.nc,C384_grid_32.0.tile2.nc,C384_grid_32.0.tile3.nc,C384_grid_32.0.tile4.nc,C384_grid_32.0.tile5.nc,C384_grid_32.0.tile6.nc
+  [ "$status" -eq 0 ]
+
+  run command make_solo_mosaic \
+                --num_tiles 6 \
+                --dir ./ \
+                --mosaic_name C384_mosaic_34.0 --tile_file C384_grid_34.0.tile1.nc,C384_grid_34.0.tile2.nc,C384_grid_34.0.tile3.nc,C384_grid_34.0.tile4.nc,C384_grid_34.0.tile5.nc,C384_grid_34.0.tile6.nc
+  [ "$status" -eq 0 ]
+
+  run command make_solo_mosaic \
+                --num_tiles 6 \
+                --dir ./ \
+                --mosaic_name C384_mosaic_35.4 --tile_file C384_grid_35.4.tile1.nc,C384_grid_35.4.tile2.nc,C384_grid_35.4.tile3.nc,C384_grid_35.4.tile4.nc,C384_grid_35.4.tile5.nc,C384_grid_35.4.tile6.nc
+  [ "$status" -eq 0 ]
+
+# stretched grid lats 32.0 34.0 35.4
+  result_32_0="$(fregrid \
+                --input_mosaic C384_mosaic_32.0.nc \
+                --input_file out \
+                --scalar_field o3 \
+                --nlon 640 \
+                --nlat 400 \
+                --latBegin 15.0 \
+                --latEnd 65.0 \
+                --lonBegin 230.0 \
+                --lonEnd 310.0 \
+                --remap_file fregrid_remap_file_640_by_400_32.0.nc \
+                --output_file out_32.0.nc --check_conserve)"
+  declare -f conserve_ratio_32_0 = echo $result_32_0 | head -n 5 | rev | cut -c39-49 | rev
+
+  result_34_0="$(fregrid \
+                --input_mosaic C384_mosaic_34.0.nc \
+                --input_file out \
+                --scalar_field o3 \
+                --nlon 640 \
+                --nlat 400 \
+                --latBegin 15.0 \
+                --latEnd 65.0 \
+                --lonBegin 230.0 \
+                --lonEnd 310.0 \
+                --remap_file fregrid_remap_file_640_by_400_34.0.nc \
+                --output_file out_34.0.nc --check_conserve)"
+  declare -f conserve_ratio_34_0 = echo $result_34_0 | head -n 5 | rev | cut -c39-49 | rev
+
+  result_35_4="$(fregrid \
+                --input_mosaic C384_mosaic_35.4.nc \
+                --input_file out \
+                --scalar_field o3 \
+                --nlon 640 \
+                --nlat 400 \
+                --latBegin 15.0 \
+                --latEnd 65.0 \
+                --lonBegin 230.0 \
+                --lonEnd 310.0 \
+                --remap_file fregrid_remap_file_640_by_400_35.4.nc \
+                --output_file out_35.4.nc --check_conserve)"
+  declare -f conserve_ratio_35_4 = echo $result_35_4 | head -n 5 | rev | cut -c39-49 | rev
+ 
+  [[ ${conserve_ratio_32_0} < 1. ]]
+  [[ ${conserve_ratio_34_0} < 1. ]]
+  [[ ${conserve_ratio_35_4} < 1. ]]
```

Comment:
Are you sure you're parsing the conservation number correctly? I don't see how the fregrid conservation information is getting to this variable. If the $conserve variables are unset, the test for comparing to 1 will pass even though it shouldn't.

Reply:
I just updated the conservation check to a better version of it. I tested several time and it works.
Please feel free to test it from your end and let me know.

I also updated the python script to use random numbers.

Thank you for your patience.